cves/CVE-2009-2902.yml

```diff
@@ -43,15 +43,14 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: This vulnerability allows a user over a network to create a specially
-  crafted WAR (Web Application Resource, including JAR files, JavaServer pages, static files, etc.)
-  file that could be deployed by an authorized user. The WAR could
-  potentially overwrite files on the system running Apache Tomcat under the
-  "work/<engine-name>/<hostname>" directory. This vulnerability is an example of
-  improper limitation of a pathname to a restricted directory because a user without
-  access to the system can violate the integrity of files they should not have access
-  to.
-
+description:
+  A path traversal bug allowed a user to specify a directory
+  to search for the a file in, and would delete the specified directory
+  as part of failure cleanup if the file had an invalid filename. 
```

Comment:
The grammar for your first description paragraph feels a bit clunky and needs some rewording. things like "to search for the a file in" doesn't make much sense.

Reply:
Fixed.